app/(app)/settings/index.tsx

```diff
@@ -32,6 +35,16 @@ const SettingsPage = () => {
     [],
   )
 
+  const { mutate, isPending } = useMutation({
+    mutationFn: () => clientApi.usersControllerDeleteUser(),
+    onSuccess: async (response) => {
+      if (response.data) {
+        const responsee = await Auth.deleteUser()
```

Comment:
`responsee` - double "e" ?

There is CodeSpell Checker plugin to VS Code :)  https://bratislava.github.io/onboarding/editor-setup#extensions

Reply:
that was just for testing purposes and I didn't remove it (response was already defined two lines above 😄 ). But thanks for suggestion :)